src/link_extractors/markdown_link_extractor.rs

```diff
@@ -297,13 +344,13 @@ mod tests {
         let img = MarkupLink {
             target: "http://meritbadge.herokuapp.com/mlc".to_string(),
             line: 1,
-            column: 6,
+            column: 5,
```

Comment:
note: I think the tests for the `column` field were incorrect beforehand; prior to my PR where `column == 6`, we had `input[column] == 't'`, which isn't the start of the link, `column = 5, input[column] == 'h'` is.

In general the test case `column`s were all 1 greater than what I believe the correct `column` values should be. Please let me know if I'm misunderstanding something!